src/modules/esl-open-state/test/esl-open-state.test.ts

```diff
@@ -0,0 +1,77 @@
+import {ESLOpenState} from '../core';
+import {ESLToggleable} from '../../esl-toggleable/core';
+import {getMatchMediaMock} from '../../esl-utils/test/matchMedia.mock';
+
+// TODO: consider as default jest loader
+jest.mock('../../esl-utils/dom/ready', () => ({
+  onDocumentReady: (clb: any) => clb()
+}));
+
+describe('ESLOpenState (mixin): tests', () => {
+  const TEST_MQ = '(min-width: 500px)';
+  const twoTicks = () => Promise.resolve().then(() => Promise.resolve());
```

Comment:
It's not two ticks. It is two serial microtasks.
It would be clearer to call it "waiting for the end of the microtask queue".

Reply:
`await microtaskQueue()`
Sounds good